### ISSUE TITLE
jwtcrack vs hashcat performance

### ISSUE THREAD

**Issue by quentusrex** [ojensen5115/jwtcrack#1]
Have you checked if there is a way to have hashcat try to crack the jwt after it's been transformed/reformatted? I'd be curious how your implementation compares. Code looks good.

**Comment by ojensen5115** [comment]
I have not checked, but I expect that if hashcat can crack json web tokens, then it is likely faster. jwtcrack makes no use of your GPU, for example.

**Comment by quentusrex** [comment]
True, GPU would likely be faster to find collisions, but hashcat does allow
you to restrict the computations to the CPU.
